manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "HDK-spec-rust",
+  "description": "This app is a living specification of Holochain and its HDK.",
+  "version": "0.1",
+  "uuid": "00000000-0000-0000-0000-000000000000",
+  "dna_spec_version": "2.0",
+  "properties": {
+      "test": "test"
+  }
+}
```

Comment:
are any of these values required?

what should happen if i provide no root properties?

Reply:
Yes, they *should* all be required, except for the "test" property, but even `properties` should be there, but could be empty.